### ISSUE TITLE
Job submission reached max poll attempts of "100"

### ISSUE THREAD

**Issue by MikeBauerCA** [zowe/zowe-cli#549]
I submitted a job with the `--vasc` flag appended. It was a relatively long job. It would be nice to be able to influence the max number of poll attempts as well as the time between polls so I could make use of this functionality in a pipeline without writing my own logic for longer running jobs.

```
zowe jobs submit ds "MY.JCL(MEM)" --vasc
Command Error:
Error obtaining status for jobname "MEM" jobid "JOB00969".
Error Details: Reached max poll attempts of "100"
```

Running Zowe CLI version 6.0.0

**Comment by MikeBauerCA** [comment]
My understanding is that we have updated this so that we poll indefinitely and progressively wait longer between polls. Therefore this particular issue has been resolved. @zFernand0 @t1m0thyj could you confirm?

**Comment by zFernand0** [comment]
That's right, we currently have static intervals between (an infinite number of) polls.
Seems like a webhook-like URL implementation (#595) may be possible thanks to the [Custom Header](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zos.v2r4.izua700/IZUHPINFO_API_PutSubmitJob.htm) `X-IBM-Notification-URL` 

Should we track that enhancement in the other issue and maybe close this one?
